lib/evaluation/handlers/subjectIsHolderEvaluationHandler.ts

```diff
@@ -28,7 +27,7 @@ export class SubjectIsHolderEvaluationHandler extends AbstractEvaluationHandler
   private readonly fieldIdzInputDescriptorsIsHolderPreferred: Map<Set<string>, Set<string>>;
   private readonly allDescribedCredentialsPaths: Map<string, string>;
 
-  private credentialsSubjects: Map<string, CredentialSubject>;
+  private credentialsSubjects: Map<string, JsonpathType>;
```

Comment:
The name is a bit confusing, given the type CredentialSubject. Suggest to do a name change